src/fal/run_scripts.py

```diff
@@ -2,18 +2,22 @@
 import os
 from typing import Dict, Any, List, Union
 from dataclasses import dataclass
-from dbt.config.runtime import RuntimeConfig
 from pathlib import Path
 
-from dbt.contracts.results import NodeStatus
+from multiprocessing.dummy import Pool as ThreadPool
```

Comment:
is this right? 

"multiprocessing.dummy replicates the API of multiprocessing but is no more than a wrapper around the threading module."

Reply:
Yes, it imitates the `multiprocessing` API, which is dedicated to forks but uses threads instead.